orcid-core/src/main/java/org/orcid/core/orgs/load/io/FtpsFileDownloader.java

```diff
@@ -0,0 +1,167 @@
+package org.orcid.core.orgs.load.io;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+import org.apache.commons.net.ftp.FTP;
+import org.apache.commons.net.ftp.FTPSClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FtpsFileDownloader {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FtpsFileDownloader.class);
+
+    private String host;
+
+    private int port;
+
+    private String username;
+
+    private String password;
+
+    private String remoteFilePath;
+
+    private String localFilePath;
+
+    private FTPSClient client;
+
+    public FtpsFileDownloader() {
+        client = new FTPSClient("TLS", true);
+
+        // if (debug) {
```

Comment:
Hey @bobcaprice could you please review this? remove it if you dont need it anymore